framework/src/transfers/MultiAppCloneReporterTransfer.C

```diff
@@ -54,12 +53,24 @@ MultiAppCloneReporterTransfer::MultiAppCloneReporterTransfer(const InputParamete
                                                     _from_reporter_names)
                            : getReporterNamesHelper(_name, _to_obj_name, _from_reporter_names))
 {
+  // Deprecating direction proves fatal for this parameter for this class
+  if (isParamValid("multi_app"))
+    paramError("The multi_app parameter is no longer valid for this class, use to_multi_app");
+
+  if (isParamValid("to_multi_app"))
+    paramError("to_multi_app",
+               "Sibling or to_multiapp transfer have not been implemented for this transfer.");
```

Comment:
We should support to_multi_app and from_multi_app. We can restrict one of them as "main" if need

Reply:
But can we cast the main app as a MultiApp for the purpose of that pointer?